src/site/shared/api/auth/NoAuthState.ts

```diff
@@ -26,7 +26,7 @@ export class NoAuthState implements AuthState {
 
     public logOut(): Promise<object> {
         SetCookie(NO_AUTH_USERNAME_COOKIE, "");
-        return Promise.resolve(null);
+        return Promise.resolve(null) as unknown as Promise<object>;
```

Comment:
Uhhhh

Reply:
`Promise.resolve(null)` is type `Promise.resolve<null>` not `Promise.resolve<object>` and cannot be directly cast